docs/userguide/antora.yml

```diff
@@ -4,6 +4,8 @@ asciidoc:
     crd: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions[CRD]
     secret-generation-extension: https://github.com/servicebinding/spec/blob/master/extensions/secret-generation.md[Secret Generation Extension]
     experimental: true
+    servicebinding-title: Service Binding Operator
```

Comment:
is this not actually `operator-name`?

Reply:
It is, yes. However, this name of variable is actually used in downstream docs (@Srivaralakshmi am I correct?), so I wanted to use the same to make it easier for the downstream to consume it.

Reply:
Yes.